UnitTests/UnitTests.UWP/Providers/Test_WindowsProvider.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using CommunityToolkit.Net.Authentication;
+using CommunityToolkit.Uwp.Authentication;
+using Microsoft.Toolkit.Uwp;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+
+namespace UnitTests.UWP.Authentication
+{
+    [TestClass]
+    public class Test_WindowsProvider : VisualUITestBase
+    {
+        // Note: Authenticaiton will work without a client id, however any Graph requests will fail.
+        private const string ClientId = "";
+
+        /// <summary>
+        /// Create a new instance of the WindowsProvider and check that is has the proper default state.
+        /// </summary>
+        [TestCategory("Providers")]
+        [TestMethod]
+        public void Test_WindowsProvider_Default()
+        {
+            WindowsProvider provider = new WindowsProvider(ClientId);
```

Comment:
Looks like you need to also updated the tests with the new changes

Reply:
Good catch. Fixed!